src/fsharp/NameResolution.fs

```diff
@@ -3095,40 +3094,57 @@ let rec ResolvePartialLongIdentInModuleOrNamespace (ncenv: NameResolver) nenv is
          // Collect up the accessible values in the module, excluding the members
          (mty.AllValsAndMembers
           |> Seq.toList
-          |> List.choose (TryMkValRefInModRef modref) // if the assembly load set is incomplete and we get a None value here, then ignore the value
-          |> List.filter (fun v -> v.MemberInfo.IsNone)
-          |> List.filter (IsValUnseen ad g m >> not) 
-          |> List.map Item.Value)
+          |> List.choose (fun x -> 
```

Comment:
I don't know what to do about this - I suppose we should just improve the performance even if code readability drops a bit.  In some of these cases it feels like they would be cleaner code as list comprehensions.